docs/.vuepress/components/DoDont.vue

```diff
@@ -35,32 +35,11 @@ export default {
       type: Array,
     }
   },
-  beforeMount() {
-    this.examples.forEach((example, idx) => {
-      this.$options.components[`comp-${example.type}${idx}`] = { ...Vue.compile(`<span>${example.caption}</span>`) };
-    })
-  },
   methods: {
-    altText(ex) {
-      if (ex.alt) return ex.alt;
-      else if (!ex.alt && ex.type === 'do') return 'Do example image';
-      else return `Don't example image`;
-    },
-    typeClass(type) {
-      return {
-        'do-dont__caption--do': type === 'do',
-        'do-dont__caption--dont': type === 'dont',
-      }
-    },
-    compiled(id, template) {
-      this.$options.components[`comp-${id}`] = { ...Vue.compile(`<span>${template}</span>`) };
```

Comment:
🤷‍♀ this doesn't seem to be a vuepress thing...don't see "compiled" used this way anywhere else in this codebase or vuepress...

Reply:
It looks like I'm responsible for all of this. It was over 2 years ago so I don't remember why, but it looks like there was the intent to also allow for compiled code examples here and not just images. That part was commented out though so I guess we never used it

Reply:
https://github.com/rei/rei-cedar-docs/blob/fb2b1e56c3e483452acf71424e2aca74c7895cdd/docs/.vuepress/components/DoDont.vue#L19